platform/linux-generic/pktio/socket_xdp.c

```diff
@@ -291,6 +292,20 @@ static int sock_xdp_close(pktio_entry_t *pktio_entry)
 	return 0;
 }
 
+static int sock_xdp_stats(pktio_entry_t *pktio_entry, odp_pktio_stats_t *stats)
+{
+	memcpy(stats, &pktio_entry->s.stats, sizeof(odp_pktio_stats_t));
+
+	return 0;
+}
+
+static int sock_xdp_stats_reset(pktio_entry_t *pktio_entry)
```

Comment:
Are `pktio_entry->s.stats` cleared somewhere during `odp_pktio_open()` open?

Reply:
Running the validation tests without `odp_pktio_stats_reset()` may be tricky since extra packets generated by Linux end up to the test interfaces.

Reply:
Indeed, but if `odp_pktio_open()` is expected to reset stats (this doesn't seem to be documented at API level, maybe there should be some clarification?) this shouldn't be a problem if implementations do the reset.